tests/python/test_with_pandas.py

```diff
@@ -358,3 +416,235 @@ def test_pyarrow_type(self, DMatrixT: Type[xgb.DMatrix]) -> None:
             if y is not None:
                 np.testing.assert_allclose(m_orig.get_label(), m_etype.get_label())
                 np.testing.assert_allclose(m_etype.get_label(), y.values)
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
+class TestPandasColumnSplit:
+    @staticmethod
+    def verify_pandas():
```

Comment:
Any chance we can reuse the pandas test instead of copying the code?

Reply:
Done.